src/aggregation/collector.rs

```diff
@@ -135,10 +145,11 @@ impl AggregationSegmentCollector {
     pub fn from_agg_req_and_reader(
         agg: &Aggregations,
         reader: &SegmentReader,
+        segment_local_id: SegmentOrdinal,
```

Comment:
```suggestion
        segment_ordinal: SegmentOrdinal,
```
segment_local_id can easily be confused with SegmentId

Reply:
Why is it not SegmentId?

Reply:
> segment_local_id can easily be confused with SegmentId

Good point. Looking back, this was probably because a function parameter took in `_segment_id: SegmentOrdinal` as a remnant of a past `SegmentLocalId` => `SegmentOrdinal` rename.

> Why is it not SegmentId
As the use-case is constructing a `DocId` that can be used to retrieve a document within the context of a search.

Reply:
I would avoid ordinal addressing if it's not necessary performance wise, as this is a common source of bugs

Reply:
As a precautionary measure, doc addresses were excluded from serialized results after we discussed these concerns back in November. If it's still concerning for some reason we can explore switching to SegmentId, but I'll probably need some pointers to start.

Reply:
Ah I just saw this is part of the `Collector` trait, which may be performance relevant in other cases. In that case we leave it like that